monai/transforms/utility/array.py

```diff
@@ -317,17 +319,33 @@ def __call__(self, img) -> np.ndarray:
         """
         if isinstance(img, torch.Tensor):
             img = img.detach().cpu().numpy()  # type: ignore
+        if isinstance(img, cp_ndarray):
```

Comment:
Might be able to fix CI/CD with:

```suggestion
        if has_cp and isinstance(img, cp_ndarray):
```

and the optional import would need `cp, has_cp = optional_import("cupy")`